src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -86,8 +86,8 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetPlatformIdentifier Condition="'$(TargetPlatformIdentifier)' == ''">Windows</TargetPlatformIdentifier>
-    <TargetPlatformVersion Condition="'$(TargetPlatformVersion)' == ''">7.0</TargetPlatformVersion>
+    <TargetPlatformIdentifier Condition="'$(TargetPlatformIdentifier)' == '' and '$(_EnableDefaultWindowsPlatform)' != 'false'">Windows</TargetPlatformIdentifier>
+    <TargetPlatformVersion Condition="'$(TargetPlatformVersion)' == '' and '$(_EnableDefaultWindowsPlatform)' != 'false'">7.0</TargetPlatformVersion>
     <TargetPlatformSdkPath Condition="'$(TargetPlatformSdkPath)' == '' and '$(TargetPlatformSdkRootOverride)' != ''">$(TargetPlatformSdkRootOverride)\</TargetPlatformSdkPath>
     <TargetPlatformSdkPath Condition="'$(TargetPlatformSdkPath)' == '' and '$(TargetPlatformIdentifier)' == 'Windows' and '$(OS)' == 'Windows_NT' and '$(MSBuildRuntimeType)' != 'Core'">$([MSBuild]::GetRegistryValueFromView('HKEY_LOCAL_MACHINE\Software\Microsoft\Microsoft SDKs\Windows\v$(TargetPlatformVersion)', InstallationFolder, null, RegistryView.Registry32, RegistryView.Default))</TargetPlatformSdkPath>
     <TargetPlatformSdkPath Condition="'$(TargetPlatformSdkPath)' == ''">$([Microsoft.Build.Utilities.ToolLocationHelper]::GetPlatformSDKLocation($(TargetPlatformIdentifier), $(TargetPlatformVersion)))</TargetPlatformSdkPath>
```

Comment:
Will `GetPlatformSDKLocation` and `GetPlatformSDKDisplayName` (on line 100) handle empty parameters correctly?  Did you do a smoke test that you can still build .NET core projects with these changes and `_EnableDefaultWindowsPlatform` set to false?

Reply:
We may want/need to disable setting `UseOSWinMdReferences` on line 97.  We won't be using WinMDs at all on .NET Core.

Reply:
Yes, `TargetPlatformDisplayName` just defaults to an empty string and I'm able to build a .NET core console app. 

Would you suggest using the same property (`_EnableDefaultWindowsPlatform`) to disable `UseOSWinMdReferences`?

Reply:
Actually, I think we should probably just set `UseOSWinMdReferences` to false in the .NET SDK (possibly conditioned on .NET 5 or higher).